documentation/operators/src/nodes/nym-api.md

```diff
@@ -126,17 +126,69 @@ Initialise your API instance with:
 
 You can optionally pass a local identifier for this instance with the `--id` flag. Otherwise the ID of your instance defaults to `default`.
 
+### Enabling credential signing on your Nym API instance
+
+To engage in the Distributed Key Generation (DKG) ceremony, it's essential to transition your `nym-api` instance from its default caching mode to the active credential signing mode. This section guides you through the process of enabling credential signing
+
+#### Generate a new wallet
+
+Begin by generating a new wallet address specifically for your instance to use in credential signing mode. Utilize the `nyxd` command-line tool with the following command:
+
+```
+nyxd keys add signer
+```
+
+It's critical to securely back up the mnemonic phrase generated during this process. This mnemonic is your key to recovering the wallet in the future, so store it in a secure, offline location.
```

Comment:
I know it should be second nature / shouldn't have to mention this at all, but imo whack this in an `admonish warning` or something just to really emphasize ?